jenkins-scripts/dsl/core.dsl

```diff
@@ -0,0 +1,47 @@
+import _configs_.*
+import javaposse.jobdsl.dsl.Job
+
+def NO_TESTING = false
+def NO_BRANCHES = []
+def NO_GITHUB_PR_INTEGRATION = false
+
+def update_vcpkg_snapshot_job = job("_vcpkg_update_snapshot")
```

Comment:
Are we going to keep this name for the job? Will this substitute our existing Windows jobs?

Reply:
The job is designed only to manage the updates of the vcpkg snapshot, it is doing something we did not have before so it can not replace any of the existing scripts. Since it is an internal job to control parts of infrastructure I'm using the `_` prefix.

Reply:
Ahhh gotcha, I completely misunderstood :smile: